data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,513 @@
+
+
+mission "Undeserved Ladybug"
+	minor
+	landing
+	invisible
+	to offer
+		has "ship model: Ancient Ladybug"
+		"reputation: Hai (Unfettered)" < -1000
+		# no you can't keep the ship forever for free, nice try
+		not "Hai Legacy 2: active"
+		# to make sure you are not stranded
+		or
+			and
+				has "flagship planet attribute: shipyard"
+				"credits" >= 1500000
+			"total ships" > 1
+	on offer
+		conversation
+			branch "unfet space"
+				has "flagship planet attribute: unfettered"
+			`As you land on this planet with the Ancient Ladybug, all warnings go on; it's as if the ship is going to fall apart!`
+			`	You order the evacuation of the ship, but as soon as every member of your crew exited...`
+			action
+				take ship "Ancient Ladybug" "with outfits"
+			`	It takes off, seemingly on its own! Looks like the Unfettered are getting this ship back at any cost.`
+				decline
+			label "unfet space"
+			`You land on this Unfettered world in possession of the Ancient Ladybug, and as you do countless Unfettered head to your ship, armed as they always are.`
+			`	Except that they seem much more intimidating than usual... They must want the ship.`
+			`	One of them hails you in a private channel, "You do not deserve to fly this relic of old, and you missed your opportunity for a glorious battle to earn the right to!"`
+				to display
+					not "Hai Legacy 3: offered"
+			choice
+				`	(Let them have it.)`
+				`	(Take off, now!)`
+					depart
+			`	You exit your ship with your crew and inform the local Unfettered militia that it is theirs now. The Unfettered that were assembling around your ship happily disperse, except for one, probably the leader judging by their special weaponry of dual pulse pistols. They come to talk to you, weapons lowered, "We are happy you could see reason, and decided to hand over the ship with no resistance. As we only wanted the ship itself, and you nicely came to deliver it to us, we have decided to allocate <payment> to you for them.`
+			`	"You may even buy the outfits back at a reduced price, if you so wish.`
+				to display
+					has "flagship planet attribute: shipyard"
+			`	"This ship was not meant for you, but remember that there are other ways for you to prove yourself, such as a bringing us a Jump Drive."`
+			action
+				take ship "Ancient Ladybug"
+				payment 2000000
+
+mission "Unfettered Landing Permit Lost"
+	landing
+	invisible
+	source
+		attributes "unfettered"
+	to offer
+		has "Hai Legacy 3: done"
+		not "outfit (all): Unfettered Landing Permit"
+		not "Unfettered Jump Drive 1: offered"
+		"reputation: Hai (Unfettered)" >= 0
+	on offer
+		dialog "As you land the Unfettered tell you they have learned of the loss of the Ladybug, thereby ending your landing access and friendly relations with them until you give them a Jump Drive."
+		# since you had to have that ship as a flagship when you received the license that means that if you lose the license you lost the ship
```

Comment:
```suggestion
		# Since you had to have that ship as a flagship when you received the license that means that if you lose the license you lost the ship.
```